src/NuGet.Core/NuGet.Common/SslProtocolUtility.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Net;
+
+namespace NuGet.Common
+{
+    public static class SslProtocolUtility
```

Comment:
I'd suggest using more common name for this class, maybe NetworkProtocolUtility?